.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,17 @@
+{
+    "configurations": [
+        {
+            "name": "Linux",
+            "includePath": [
+                "${workspaceFolder}/**",
+                "/home/kalyan/Documents/pgsql-11.17/**"
+            ],
+            "defines": [],
+            "compilerPath": "/usr/bin/gcc",
+            "cStandard": "gnu17",
+            "cppStandard": "gnu++17",
+            "intelliSenseMode": "linux-gcc-x64"
+        }
+    ],
+    "version": 4
+}
```

Comment:
Always be sure, **not** to include the files with your environment details (unless required/specified)

Reply:
Thank you for rectifying it. Should I remove these files and create a new Pull Request?

Reply:
Just remove the files and commit them here, and the PR would be updated automatically. No need for creating a new PR.